docs/changelog/2024/2412.md

```diff
@@ -0,0 +1,127 @@
+---
+  title: Cycle 2412 (28 Nov)
+---
+
+--8<-- "includes/abbreviations.md"
+
+Cycle 12 is always a major update for NZZC, with this year being no different. Cycle 2412 implements Auckland DMAPs, ensuring further separation between departures and missed approach paths. There are also a plethora of changes to IFR waypoints.
+
+## SOP Changes
+
+### TBC (Pending Mikey's Changes)
```

Comment:
@VATNZ-MR Were you wanting to add some stuff into this section?

Reply:
Done!